frontend/src/pages/article/article-page.tsx

```diff
@@ -19,8 +19,12 @@ const ArticlePage: React.FC = () => {
 
   const { id } = useParams();
 
+  const [initialDataLoad, setInitialDataLoad] = useState(false);
```

Comment:
I believe we don't need this boolean flag, let's just try to fix `dataStatus` management in article slice -- for now - just create separate dataStatus for getArticle action